.github/workflows/codeql.yml

```diff
@@ -0,0 +1,49 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main", "release/*" ]
+    paths:
+      - 'checkout-3ds-sdk-android/**'
+      - 'checkout-3ds-sdk-ios/**'
+  schedule:
+    - cron: '39 10 * * 5'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+    
+jobs:
+  analyze:
+    name: Code Analyze
+    runs-on: ubuntu-latest
+    timeout-minutes: ${{ (matrix.language == 'swift' && 120) || 360 }}
```

Comment:
curious why we have different time out mins for different language? either way, 120 and 360 seems quite a long time to analysis the static web code, perhaps we could reduce it to 5 mins max?

Reply:
Make sense updated it to 5 mins